changelog/1.18.0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "1.18.0"
+description: "Released on 04/21/2021"
+---
+
+### Breaking changes ❗
+
+- web: Renamed _environments_ to _workspaces_. URLs containing _environment_
+  should redirect to use _workspace_. It is possible that some URLs will not
+  link to the expected resource.
+
+### Features ✨
+
+- web: Added the ability to specify a workspaces as code template filepath other
+  than the default _coder.yaml_
+- web: Moved _built-in_ sign-in behind a toggle so that it does not display when
+  OIDC is configured for authentication
+  - additionally, updated color scheme on the sign-in page
+- web: Added dev URL support in workspaces as code templates
+- web: Added admin toggle for workspaces as code feature
```

Comment:
probably needs docs changes

Reply:
Yep - https://app.clubhouse.io/coder/story/11187/wac-1-18-documentation-updates - we unfortunately didn't plan ahead there